modules/storefront-ui.yml

```diff
@@ -1,18 +1,20 @@
-name: storefront-ui
-description: Storefront UI Module for Nuxt
-repo: vuestorefront/storefront-ui#develop/packages/nuxt-module
+name: Storefront-ui
+description: Storefront UI for Nuxt by Alokai
+repo: vuestorefront/storefront-ui#v2-develop/packages/sfui/frameworks/nuxt
 npm: '@storefront-ui/nuxt'
 icon: storefront-ui.svg
-github: >-
-  https://github.com/vuestorefront/storefront-ui/tree/develop/packages/nuxt-module
-website: >-
-  https://docs.storefrontui.io/?path=/story/getting-started-installation--page#installation
-learn_more: https://docs.storefrontui.io/
+github: https://github.com/vuestorefront/storefront-ui
+website: https://docs.storefrontui.io/v2/
+learn_more: https://docs.storefrontui.io/v2/getting-started/vue#nuxt-3
 category: UI
 type: 3rd-party
 maintainers:
-  - name: Jakub Andrzejewski
-    github: Baroshem
+  - name: Szymon Dziewoński
+    github: Szymon-dziewonski
+    avatar: https://avatars.githubusercontent.com/u/2566152?v=4
+  - name: Jakub Freisler
+    github: FRSgit
+    avatar: https://avatars.githubusercontent.com/u/10456649?v=4
 compatibility:
-  nuxt: ^2.0.0
```

Comment:
is there still a Nuxt v2 compatible verison?

Reply:
I don't it is, at least no one checked if it is working on nuxt v2. Also I think this file I edited is not available in search anymore https://nuxt.com/modules that's why I modified it, @Baroshem @danielroe can you please confirm?